src/openfluid/ui/waresdev/FragmentCreationDialog.cpp

```diff
@@ -128,7 +128,10 @@ void FragmentCreationDialog::createFragment()
   // Create file if wanted
   if (ui->createFileCheckBox->isChecked())
   {
-    FragmentsPath.makeFile(FragmentID+".hpp");
+    openfluid::tools::FilesystemPath FragmentFilePath({FragmentsPath.toNative(),
+                                                       FragmentID+".hpp"});
+    FragmentsPath.makeFile(FragmentFilePath.toNative());
```

Comment:
Prefer use toGeneric() instead. toNative() is mainly dedicated to display paths.

Reply:
The optional path given to makeFile() method should be relative to the calling path object. Maybe a potential error here?